### PR TITLE
Ticker subsystem cleanup

### PR DESCRIPTION
**Pull request by AnturK**
Need to test if this clockcult stuff is not too late.

### CODE REVIEW

File: code/modules/antagonists/clockcult/clockcult.dm [hunk 1]
```diff
@@ -193,6 +193,17 @@
 	var/list/objective
 	var/datum/mind/eminence
 
+/datum/team/clockcult/New(starting_members)
+	. = ..()
+	START_PROCESSING(SSobj,src)
```
Comment (ninjanomnom) [inline code/modules/antagonists/clockcult/clockcult.dm]: Why don't we have a gamemode subsystem or something

Comment (AnturK) [reply code/modules/antagonists/clockcult/clockcult.dm]: Mostly because only things that could use is this and rev headcount maybe.

**Comment by Cyberboss** [comment]
honk

**Comment by AnturK** [comment]
This part works fine, testing made me realize creating clock cult mid round is a pain due toe cog z level though.

File: code/modules/antagonists/clockcult/clockcult.dm (hunk 1 above)
Comment (optimumtact) [reply code/modules/antagonists/clockcult/clockcult.dm]: also SSticker is the gamemode subsystem

**Pull request merged**